modules/schematics/src/effect/index.spec.ts

```diff
@@ -463,4 +464,35 @@ describe('Effect Schematic', () => {
 
     expect(content).toMatch(/effects = TestBed\.inject\(FooEffects\);/);
   });
+
+  it('should add prefix to the effect', async () => {
```

Comment:
One small remark, could you also write a test for when the `action creators` option is `true` please, because that's the one that is most used. The rest is looking good  👍

Reply:
@timdeschryver , added a test with creators option set as `true` for in the effect spec file. I'm not sure but do we need tests with `creators: true` in `feature` and `reducer` as well?

Reply:
The `feature` and `reducer` tests are by default `creators: true`, so this is fine for me.
Thanks for updaing the PR! 👍